### DISCUSSION TITLE
ccf-validation-tools/Ovary/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#325] [General]
# ccf-validation-tools/Ovary/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Ovary/comments/

**Comment by emquardokus** [comment]
Ovary v1.3 https://docs.google.com/spreadsheets/d/1tk0zBaCqaUXn96uhcgAiUTbO03zYnI-0DQ9OmHdxFQQ/edit#gid=756296951
**Typos or punctuation mistakes**

Removed EFO term and put into NOTES to temporarily fix issue
Cell ontology issue opened April 26, 2024 https://github.com/obophenotype/cell-ontology/issues/2360

    It might have a typo in the term EFO:0005289 in the following 72 rows 273, 286, 299, 312, 325, 338, 351, 364, 377, 390, 403, 416, 441, 442, 443, 444, 445, 460, 461, 462, 463, 464, 479, 480, 481, 482, 483, 498, 499, 500, 501, 502, 517, 518, 519, 520, 521, 536, 537, 538, 539, 540, 555, 556, 557, 558, 559, 574, 575, 576, 577, 578, 593, 594, 595, 596, 597, 612, 613, 614, 615, 616, 631, 632, 633, 634, 635, 650, 651, 652, 653, 654. The term id should have this pattern: UBERON:NNNNNNN or CL:NNNNNNN or PCL:NNNNNNN. The ontology name in upper case. N is a number, and it should have exactly 7 numbers after the colon. Please change it in the ASCT+B table.

**Additional fixes**
Ref/n/DOI columns all changed to new format Ref/n/ID
All citations updated to agreed upon format with support for all types of citations, not just DOI.

**Comment by aleixpuigb** [comment]
New terms for ovary in release v2024-07-10:

Ovary: pre-granulosa cell (CL:4033066)
Ovary: mural granulosa cell (CL:4033067)

**Comment by aleixpuigb** [comment]
**New cell terms for ovary:**

Ovary: squamous granulosa cell (CL:4033083)
Ovary: cuboidal granulosa cell (CL:4033084)
Ovary: pre-theca cell (CL:4052010)
